### ISSUE TITLE
Beta 7 on Mac 10.9.5 issues

### ISSUE THREAD

**Issue by steven26** [apohl79/audiogridder#174]
Dear sir,

I installed beta 7 server on Mac 10.9.5.
Screen capture is set to FFmpeg.
If you open a plugin in screen in the DAW (Ablteton) the server restarts.
Than in the dock there are two Audiogridder server icons, soon the same thing happens again and than you
have three Audiogridder icons in the dock and so on.
You can close the second and third (and next) Audiogridder instances by right clicking the icon in the dock and select quit.
You can not close the first instance, it does not close even if you select "force quit".
The instance is not shown in the activity monitor and you can not shut down you computer anymore, unless pressing the off button for some thine to force a shut down.

There is no crash report.

If you disable screen capture the server is working slightly better, but the issue happens also, although it takes longer before  it happens (in that case you change the settings of a plugin on the remote computer).

Any ideas how to solve this issue?

Kind regards,

Steven

**Comment by apohl79** [comment]
Can you check the server logs? I would expect the master log to report servers to not shutdown properly. So it restarts the server process. The server logs might show some errors. Maybe some plugin does not work. Usually the server would never shutdown, unless you quit it, so not sure why there is no crash report.

Do you also have multiple tray icons?

When this happens, can you open the OSX activity monitor and check if there are multiple AG server processes? There should be only 2. If there are more, are there processes that do not respond (usually red)? if yes, can you generate a spindump for such a process (mark the process row, click on the settings icon and select spindump in the dropdown)?

**Comment by steven26** [comment]
Thanks for your reply.

This issue is happening with all plugins.
There is only one tray icon, that is white (on another macbook the server icon is black).

When there are more Audiogridder icons in the dock, there is only one Audiogridder entry in the activity monitor.
I can close the additional Audiogidder icons in the dock by right clicking and select quit.
However i can not close the first icon and there is no process for that entry in the activity monitor.

Can you tell me where i can find the server log so i can send that?

Kind regards,

Steven

**Comment by apohl79** [comment]
Have a look at this page: https://audiogridder.com/bug-reports/

I never tested older os x versions. I hope it is no incompatibility issue with 10.9. I know some people were using 10.7. 

Maybe you could also try beta8. it has more logging functionality (tracing mode).

**Comment by steven26** [comment]
Thanks for your reply.

I have attached 4 log files (2 crash log files and 2 server log files):

[crash_1_log.txt](https://github.com/apohl79/audiogridder/files/5364665/crash_1_log.txt)
[crash_2_log.txt](https://github.com/apohl79/audiogridder/files/5364671/crash_2_log.txt)
[log_1.txt](https://github.com/apohl79/audiogridder/files/5364672/log_1.txt)
[log_2.txt](https://github.com/apohl79/audiogridder/files/5364674/log_2.txt)

I will try beta 8 tonight.

Kind regards,

Steven

**Comment by apohl79** [comment]
OK great! The two crash logs show that the crash was caused by the FFMpeg code (will have a closer look later). You said it also crashes w/o ffmpeg. Can you try that too and provide a crash log? Thx!

**Comment by apohl79** [comment]
You can also try the legacy capturing mode. It is not so efficient but might be more stable.

**Comment by steven26** [comment]
Thanks for your reply.

From what i can remember, even with screen capture disabled i was still able to open the plugin screen on the DAW.
I can not try this right away, but i will try again with capture disabled and legacy mode tonight ans let you know the outcome.

Kind regards,

Steven

**Comment by steven26** [comment]
I have tried the legacy capturing mode.
That seems to be working without any issues on a Macbook with OS 10.9.5.

I have tested this with Acustica Audio Modula.
The goal was to be able to run Modula on a remote computer, and i can add about 16 channel strips with acceptable load/latency on the Macbook (2012, I7).
The screen capture is taking quite some system resources (+/- 15%), but that is the same when using FFMpeg on another Macbook.

It's a bit strange that after saving a project in Ableton, close the project and re-open the project, the fader order in Modula is different than before closing the project.
I guess this depend on the order Audiogridder is added (or adds the plugins) when opening a project.

I like your project (I will make a donation tomorrow) and think when just using it for plugins, it is better than VIenna Ensamble. A cooperation with Acustica Audio would be great.

Kind regards,

Steven
